sys-auth/libfprint/libfprint-1.90.7.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson udev
+
+DESCRIPTION="Library to add support for consumer fingerprint readers"
+HOMEPAGE="https://cgit.freedesktop.org/libfprint/libfprint/ https://github.com/freedesktop/libfprint https://gitlab.freedesktop.org/libfprint/libfprint"
+SRC_URI="https://github.com/freedesktop/libfprint/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="examples gtk-doc +introspection"
+
+RDEPEND="
+	dev-libs/glib:2
+	>=dev-libs/libgusb-0.3.1
+	dev-libs/nss
+	virtual/libusb:1=
+	x11-libs/gtk+:3
+	x11-libs/libX11
+	x11-libs/libXv
+	x11-libs/pixman
+	!>=sys-auth/libfprint-1.90:0
+"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	virtual/pkgconfig
+	gtk-doc? ( dev-util/gtk-doc )
+	introspection? ( dev-libs/gobject-introspection )
+"
+
+PATCHES=( ${FILESDIR}/${PN}-0.8.2-fix-implicit-declaration.patch )
+
+src_configure() {
+		local emesonargs=(
+			$(meson_use examples gtk-examples)
+			$(meson_use introspection)
+			-Dgtk_doc=$(usex gtk-doc true false)
```

Comment:
Sorry I forgot to mention, but you can use `meson_use` here. Check how examples are installed above.

```
WARNING: Unknown options: "gtk_doc"
...
Program gtkdoc-scan found: NO
```
https://github.com/freedesktop/libfprint/blob/7c2a67a9544dd5b7bfa134ad7add566590f9c913/meson_options.txt

Reply:
I'm sorry for the mistake, fixed now